crates/valence_ident/Cargo.toml

```
@@ -1,5 +1,7 @@
 [package]
 name = "valence_ident"
+description = "Parse and create Minecraft's resource locations"
```

```suggestion
description = "Parse and create Minecraft's namespaces"
```
I'm maybe wrong but this terminology seems more common around users. (I personally never had ear of resource location before and yes I know on the wiki it's resource location)

---

If we're nitpicking, it would probably make more sense to call them "resource identifiers", similar to the URI acronym.